spec/features/home_map_spec.rb

```diff
@@ -107,7 +107,6 @@ def click_first_map_marker
     expect(page).to have_no_css('#filterResultsTrigger')
     expect(page).to have_css('#filterClose')
     expect(page).to have_content('3 facility matches (of 3)')
-    expect(page).to have_content('4 practices matched (of 4)')
```

Comment:
Shouldn't we keep this and just change it to `innovations`?

Reply:
yes! I think it's because when we are filtering we aren't actually changing the word to say innovations @bradjohnson92008 see `app/assets/javascripts/diffusion_history/home_map.es6` line 112